sdl2-mixer/PSPBUILD

```diff
@@ -30,9 +40,13 @@ build() {
 }
 
 package() {
+    mkdir -p "$pkgdir/psp/sdk/samples/sdl2/mixer/"
+    mv main.c "$pkgdir/psp/sdk/samples/sdl2/mixer/"
```

Comment:
I would use the `install` command instead of `mv`. Make sure to get the files from `$srcdir/`. Right now these files are 0 byte links instead of the actual files.